backend/src/v2/component/launcher_v2.go

```diff
@@ -507,6 +516,26 @@ func execute(
 		writer = os.Stdout
 	}
 
+	// If a custom CA path is input, append to system CA and save to a temp file for executor access.
+	if customCAPath != "" {
+		if caBundleTmpPath, err := compileTempCABundleWithCustomCA(customCAPath); err != nil {
+			return nil, err
+		} else {
```

Comment:
Optional: You can remove the else since you always return inside the if. It makes it easier to read.